### DISCUSSION TITLE
Disabling automatic argument names

### DISCUSSION THREAD

**Discussion by tikatoo** [LazyVim/LazyVim#3635] [Q&A] [answered]
Heya, so about a week or so ago LazyVim started inserting argument names in function calls (see below). It's a neat idea, but after a few days it's starting to feel a bit intrusive - so how would I go about turning it off?

![Screenshot from 2024-06-14 15-43-27](https://github.com/LazyVim/LazyVim/assets/48166618/528f1d4e-40b1-43b8-af27-4aca866cb555)

Unfortunately, I've been unable to find anything about this in either the LazyVim documentation (minimal as it is) or in the recent changelogs. Maybe that's just because I don't actually know what the feature is called, or which plugin it could possibly be from.

**Comment by dpetka2001** [comment, marked as answer]
It's called `inlay_hints` and is provided directly from the LSP servers and Neovim. You can press `<leader>uh` to toggle them on/off only for you current session or for Neovim to start without this feature enabled do
```lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      inlay_hints = { enabled = false },
    },
  }
}
```

**Reply by tikatoo** [reply]
Ah yes, that's exactly it! You're a gem, thank you :blue_heart:
